conduct.md

```diff
@@ -110,9 +110,9 @@ TODO: Write an introductory paragraph.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should avoid omitting failures.
```

Comment:
I agree with @Ochirsaikhan that it would be good to elaborate on what exactly "omitting failures" looks like!

Reply:
I just added a bit more detail to that point to see if that's more clear now.